doc/getting_started.md

```diff
@@ -0,0 +1,14 @@
+Getting Started
+===============
+
+This package implements a RobotDriver and several applications for the TriFinger
+robots, using the [`robot_interfaces`
+package](https://open-dynamic-robot-initiative.github.io/code_documentation/robot_interfaces/docs/doxygen/html/index.html).
+See there for more information on the general architecture.  The action and
+observation types for the (Tri)Finger robots are also implemented in
+`robot_interfaces`.
+
+For a basic example on how to control the robot, see
+[demo_real_finger.py](https://github.com/open-dynamic-robot-initiative/robot_fingers/blob/master/demos/demo_real_finger.py)
```

Comment:
this may be a little confusing, because this demo is for only one finger. we could remark this here, or point to the trifinger demo